docs/docs/deployments.md

```diff
@@ -1,312 +0,0 @@
----
```

Comment:
This file should have been removed some time back as it's been moved into parts under /deploy